cli/commands.go

```diff
@@ -33,15 +33,15 @@ easy-to-read format.`[1:],
 			if err != nil {
 				return codeErr{err, 2}
 			}
-			if err := run.Preview(env, cfg, useYaml); err != nil {
+			if err := run.Preview(env, cfg, format); err != nil {
```

Comment:
we should check if the format is a valid one that we know about here, and exit with an error code of 2 if not.